Apps/Src/Tasks.c

```diff
@@ -6,6 +6,10 @@
 #include "Voltage.h"
 #include "Temperature.h"
 
+#include "../../BSP/Inc/BSP_Lights.h"
```

Comment:
Why are you including these here?

Reply:
So I can change the LED's that are on. It also lets me turn on the strobe light.

Reply:
Do you use these anywhere in this file though? It looks like your only changes here are to include more files and to add commented out code

Reply:
I thought this was the faultState file, not tasks file. I'll remove it.

Reply:
I just checked the commit history and Ethan made all those changes to tasks.c so I'm just going to undo all of them.

Reply:
Ok, sounds good